src/app/app.module.ts

```diff
@@ -8,13 +8,17 @@ import { MaterialModule } from './shared/material.module';
 import { HeaderComponent } from './shared/ui-material/header/header.component';
 import { FooterComponent } from './shared/ui-material/footer/footer.component';
 import { HomePageComponent } from './home-page/home-page.component';
+import { HeroSectionComponent } from './hero-section/hero-section.component';
+import { ProductsComponent } from './products/products.component';
 
 @NgModule({
   declarations: [
     AppComponent,
     HeaderComponent,
```

Comment:
create a separate module for shared components in shared folder.
move header and footer components to a shared folder